### PR TITLE
Tweak bash installer + fix releasing

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: doc/getting-started/installation.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 > This is the recommended way to install Castor on Linux and macOS. It requires PHP >= 8.1.
 
 ```bash
-curl "https://raw.githubusercontent.com/jolicode/castor/main/installer/bash-installer" | bash
+curl "https://jolicode.castor.com/install" | bash
 ```
 
 ### As a phar
```
Comment (lyrixx) [inline doc/getting-started/installation.md:14]: I would move this later in the page. WDYT?

File: doc/getting-started/installation.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 > This is the recommended way to install Castor on Linux and macOS. It requires PHP >= 8.1.
 
 ```bash
-curl "https://raw.githubusercontent.com/jolicode/castor/main/installer/bash-installer" | bash
+curl "https://jolicode.castor.com/install" | bash
```
Comment (pyrech) [inline doc/getting-started/installation.md:11]:
```suggestion
curl "https://castor.jolicode.com/install" | bash
```
I assume you didn't test this 😏🤭

File: doc/getting-started/installation.md (hunk 1 above)
Comment (pyrech) [reply doc/getting-started/installation.md:14]: This also means the recommended installation for windows is move to the bottom.

File: doc/getting-started/installation.md [hunk 3]
```diff
@@ -34,8 +38,9 @@ curl "https://github.com/jolicode/castor/releases/latest/download/castor.linux-a
     (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
 ```
 
-> **Note:** Castor is also released for Linux running on ARM64 architectures.
-> In such case, replace `linux-amd64` with `linux-arm64` in the URL.
+> [!NOTE]
```
Comment (pyrech) [inline doc/getting-started/installation.md:41]: I started to make a sub chapter like it's done for mac, but anyway, let's keep it like that for now

File: src/Listener/UpdateCastorListener.php [hunk 4]
```diff
@@ -132,7 +132,7 @@ private function displayUpdateWarningIfNeeded(InputInterface $input, OutputInter
 
             if (OsHelper::isUnix()) {
                 $symfonyStyle->block('Run the following command to update Castor:');
-                $symfonyStyle->block(sprintf('<comment>curl "%s" -Lfso castor && chmod u+x castor && mv castor %s</comment>', $latestReleaseUrl, $pharPath), escape: false);
+                $symfonyStyle->block(sprintf('<comment>curl "https://jolicode.castor.com/install" | bash --install-dir %s</comment>', $pharPath), escape: false);
```
Comment (pyrech) [inline src/Listener/UpdateCastorListener.php:135]:
```suggestion
                $symfonyStyle->block(sprintf('<comment>curl "https://castor.jolicode.com/install" | bash --install-dir %s</comment>', $pharPath), escape: false);
```

**Review by pyrech — COMMENTED** [review]
I wonder whether we should have instructions with selectors for OS /installation modes to directly display the right command  to the users. But that's for later.

File: doc/getting-started/installation.md (hunk 2 above)
Comment (lyrixx) [reply doc/getting-started/installation.md:11]: Indeed :) it's not yet deployed 😅

**Comment by lyrixx** [comment]
I changed a bit the installation order chapter: See it live: https://lyrixx.github.io/castor-test/getting-started/installation/

**Review by pyrech — APPROVED** [review]

**Pull request merged**